golem_messages/factories/tasks.py

```diff
@@ -99,7 +159,18 @@ def task_id(
         if ttc.compute_task_def is None:
             return
 
-        ttc.compute_task_def['task_id'] = extracted or helpers.fake_golem_uuid(  # noqa pylint: disable=unsupported-assignment-operation
+        _task_id = None
+        with suppress(AttributeError):
+            if ttc.requestor_id == \
+                    ttc.want_to_compute_task.task_header.task_owner.key:
+                _task_id = ttc.want_to_compute_task.task_header.task_id
+        # except AttributeError:
+        #     pass
```

Comment:
If you left this comments it only confirms that *suppress* is not bvious way of silencing exceptions.

Reply:
no, it's just me being inattentive... sorry, I will remove it